.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
   - conda update conda --yes
-  - conda create -n testenv --yes pip nose python=$TRAVIS_PYTHON_VERSION pymongo six pyyaml numpy h5py pims jsonschema
+  - conda create -n testenv --yes pip nose python=$TRAVIS_PYTHON_VERSION pymongo=2.8 six pyyaml numpy h5py pims jsonschema
```

Comment:
@tacaswell says that this broke for him on 2.9, apparently due to Travis weirdness not an actual problem. He wants to punt for now, and I want this merged so I can use it.